.cbmc-batch/jobs/aws_cryptosdk_rsa_decrypt/Makefile

```diff
@@ -44,6 +42,13 @@ DEPENDENCIES += $(SRCDIR)/helper-src/openssl/ec_override.goto
 DEPENDENCIES += $(SRCDIR)/helper-src/openssl/err_override.goto
 DEPENDENCIES += $(SRCDIR)/helper-src/openssl/evp_override.goto
 
+ENTRY = aws_cryptosdk_rsa_decrypt_harness
+
+REMOVE_FUNCTION_BODIES += --remove-function-body EVP_PKEY_FREE
+REMOVE_FUNCTION_BODIES += --remove-function-body evp_pkey_is_valid
```

Comment:
why?

Reply:
I was mistakenly under the impression this was necessary when using stubs, but I have corrected this and added a comment about why the abstractions are used.